src/scripts/rank_SVs.pl

```diff
@@ -297,15 +297,17 @@ sub scoring {
 	my $debug = 0;
 	my $medal = 0;
 	my $rating = 'LQ';
-	$medal = 1 if(exists($known_fusion_partners{$fg1}));
-	$medal = 1 if(exists($known_fusion_partners{$fg2}) && $sv->{ort} eq "?");
-	$medal = 2 if(exists($known_fusion_partners{$fg2}) && $sv->{ort} eq ">");
-	$medal = 3 if(exists($known_fusion_partners{$fg1}) && exists($known_fusion_partners{$fg2}) && $sv->{type} !~ /Internal/);
-	$medal = 4 if(exists($known_fusions{$fusion}));
-	
-	if(exists($known_ITDs{$fg1}) && $type eq 'Internal_dup' &&
-	   ($bp1->{tpos} - $known_ITDs{$fg1}[0]) * ($bp1->{tpos} < $known_ITDs{$fg1}[1]) > 0 &&
-	   ($bp2->{tpos} - $known_ITDs{$fg1}[0]) * ($bp2->{tpos} < $known_ITDs{$fg1}[1]) > 0){
+
+	$medal = 1 if(exist_multiplename_checking(\%known_fusion_partners,$fg1));
+        $medal = 1 if(exist_multiplename_checking(\%known_fusion_partners,$fg2) && $sv->{ort} eq "?");
+        $medal = 2 if(exist_multiplename_checking(\%known_fusion_partners,$fg2) && $sv->{ort} eq ">");
+        $medal = 3 if(exist_multiplename_checking(\%known_fusion_partners,$fg1) && exist_multiplename_checking(\%known_fusion_partners,$fg2) && $sv->{type} !~ /Internal/);
+        $medal = 4 if(exists_knownfusionlist_checking(\%known_fusions,$fg1,$fg2));
+
+	my ($Is_known_ITD, $ITD_left_coor, $ITD_rightt_coor)=match_known_ITD(\%known_ITDs,$fg1);	
+	if($Is_known_ITD && $type eq 'Internal_dup' &&
+	   ($bp1->{tpos} - $ITD_left_coor) * ($bp1->{tpos} < $ITD_rightt_coor) > 0 &&
+	   ($bp2->{tpos} - $ITD_left_coor) * ($bp2->{tpos} < $ITD_rightt_coor) > 0){
```

Comment:
Are the inequalities (e.g., `$bp1->{tpos} < $ITD_rightt_coor`) for arithmetic intentional here?